airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialReadUtil.java

```diff
@@ -179,6 +179,7 @@ public static InitialLoadStreams streamsForInitialPrimaryKeyLoad(final CdcStateM
                                                                    final ConfiguredAirbyteCatalog fullCatalog,
                                                                    final boolean savedOffsetStillPresentOnServer) {
     if (!savedOffsetStillPresentOnServer) {
+      LOGGER.info("FULL RESYNC DUE TO SAVED OFFSET ISSUE");
```

Comment:
checking: intentionally left this log?

Reply:
No, I need to revert this file and `MySqlInitialStateIterator.java`. Thanks for checking!